nx/lib/nx/binary_backend/matrix.ex

```diff
@@ -849,7 +856,7 @@ defmodule Nx.BinaryBackend.Matrix do
       |> Enum.map(fn col ->
         row
         |> Enum.zip(col)
-        |> Enum.reduce(0, fn {x, y}, acc -> acc + x * y end)
+        |> Enum.reduce(0, fn {x, y}, acc -> acc + Complex.multiply(x, y) end)
```

Comment:
```suggestion
        |> Enum.reduce(0, fn {x, y}, acc -> acc + x * Complex.conjugate(y) end)
```

Reply:
The process here is correct to multiply complex elements (x * y), but the process is different from the (x*y^*) suggested. I misunderstood and committed the suggestion once myself, but I reverted to the original code. [56e6662](https://github.com/elixir-nx/nx/pull/933/commits/56e66629ad7cdcf22c07230a6793947713e858c5)